vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -40,7 +44,7 @@ def save_inventory(type, parent, hash)
     found = parent.send(type)
     if found.nil?
       found = parent.send("build_#{type}", hash.except(:id))
-      parent.send("#{type}=", found)
+      parent.update_attributes!(type => found)
```

Comment:
Also, do we have to change https://github.com/ManageIQ/manageiq/blob/28b598ed5485a31a8d617b45c00856d23bb7dbcc/vmdb/app/models/ems_refresh/save_inventory_helper.rb#L53-55 to be consistent?  I don't understand why this method was added in the first place since it's so similar to save_inventory.  @kbrock Can you comment on this?